firefox_ui_harness/testcases/__init__.py

```diff
@@ -0,0 +1,5 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from update import *
```

Comment:
Yes, that's part of the cleanup story I mentioned yesterday. With all the code moving around I wanted to have it as easy as possible.